src/parsing/Parse_pattern.mli

```diff
@@ -1,2 +1,9 @@
-val parse_pattern : ?print_errors:bool -> Lang.t -> string -> Pattern.t
-val parse_pattern_ref : (bool -> Lang.t -> string -> Pattern.t) ref
+val parse_pattern :
+  ?print_errors:bool ->
+  ?rule_options:Rule_options_t.t option ->
```

Comment:
here it's ok to depend on Rule_options_t. We are inside semgrep code. Languages/ might be used by the OCaml community at some point, so we don't want to add extra dependency to semgrep-specific stuff.

Reply:
Done